topostats/io.py

```diff
@@ -32,6 +33,48 @@
 # pylint: disable=too-many-lines
 
 
+def dict_almost_equal(dict1, dict2, abs_tol=1e-9):
```

Comment:
There are solutions to comparing dictionaries but not recursively, with numpy arrays and allowing a tolerance as far as I can tell.

Reply:
Yes, I found similar classes/functions mentioned in various places.

I think as this is solely for testing it perhaps shouldn't be in `topostats.io` but defined in the test files where its compared. Not sure if that is good practice or if it could be incorporated in some other manner.